mubench.reviewsite/src/Controllers/SnippetsController.php

```diff
@@ -34,8 +35,8 @@ public function deleteSnippet(Request $request, Response $response, array $args)
         return $response->withRedirect($form['path']);
     }
 
-    static function createSnippetIfNotExists($projectId, $versionId, $misuseId, $file, $line, $code)
+    static function createSnippetIfNotExists($projectId, $versionId, $misuseId, $file, $line, $code, $detectorId = null)
```

Comment:
This delegate method doesn't really add anything. Please replace it with the `Snippet::createIfNotExists()` method.